content/resources/gen3-features.md

```diff
@@ -0,0 +1,81 @@
+---
+identifier: gen3-features
+title: Gen3 features
+linktitle: /resources/gen3-features
+date: 2019-12-04T10:57:21-05:00
+---
+
+
+# A list of Gen3 features
+
+Gen3 is an open-source platform that enables the standing-up of data commons for managing, analyzing, and sharing research data. Below you can find an exhaustive list of Gen3 features.
+
+## Dictionary and metadata
+
+* Explore the graph in the dictionary page of Windmill:
+    * Search keywords
+    * Visualize possible paths between nodes
+    * Download JSON or TSV format for node templates 
+* Explorer project graph and nodes in Windmill:
+    * Visulize graph model
+    * Browse/download data for each node
+* Metadata submission (create, update, delete records) in JSON, TSV or CSV:
+    * Submission page of Windmill
+    * Gen3 SDK (Python, R)
+* Export metadata as a JSON, TSV or CSV file
+* The metadata is stored in Postgres databases
+* ElasticSearch:
+    * The metadata can be stored in ElasticSearch by using our ETL tool
+    * Explore the ElasticSearch metadata using Guppy
+* Generate fake data specific to a dictionary for testing
+* Import and export using the Portable Format for Biomedical Data (PFB): *(Work in progress)*
+    * Import and export a dictionary
+    * Import and export metadata
+
+## Data files
+
+* Data upload through the Gen3 Client
+* Indexing data files:
+    * Create data file records and mint GUIDs
+    * Index a new version of the same file
+    * Get the most recent version of a file
+    * Index data files that are stored outside of the Commons
+* GUID resolver
+* Data download:
+    * Gen3 Client
+    * Files page of Windmill
+* Core metadata endpoint to get information about a data file
```

Comment:
Out of curiosity, what is this? Is this the indexd endpoint? Or something else?

Reply:
Yes, indexd endpoint

Reply:
this is the pidgin endpoint

Reply:
it should be called "core metadata" because that's what is used in the pidgin docs. it uses the core_metadata_collection node. so renaming it "core structured data" does not work

Reply:
Thank you for the comment, fixed in the next commit.